### ISSUE TITLE
Use desktop fullscreen mode instead of changing resolution

### ISSUE THREAD

**Issue by danhedron** [rwengine/openrw#231]
GameWindow currently uses `SDL_WINDOW_FULLSCREEN` instead of `SDL_WINDOW_FULLSCREEN_DESKTOP`, which will change the video mode.

This could be done by using `SDL_WINDOW_FULLSCREEN_DESKTOP` to get a desktop sized fullscreen window and upsampling from the requested resolution to the desktop resolution.

**Comment by ShFil119** [comment]
Does #475 cover it?

**Comment by danhedron** [comment]
No. Fixing this would involve using the ``SDL_WINDOW_FULLSCREEN_DESKTOP`` mode and rendering at the requested window resolution instead.
